insights/config/__init__.py

```diff
@@ -588,6 +588,16 @@ def __str__(self):
     def __init__(self, command, multi_output=True, large_content=False, **kwargs):
         if command[0] != '/':
             raise SpecPathError(command, "CommandSpec command must start with '/'")
+        for shellism in ('&&', '||', '|', '>', '<'):
+            # The index will always work since we know from the previous test
+            # that index 0 is not one of these shellisms (because it's '/').
+            if shellism in command and command[command.index(shellism) - 1] == ' ':
+                raise SpecPathError(command,
```

Comment:
Since this is not really a path exception, should you create a new exception class for this such as `SpecSyntaxError`?  

Also this is failing the tests because the `SpecPathError` exception does not have a `__str__` method.  If you add something like this to the `SpecPathError` class then it should work correctly.
```
def __str__(self):
    return '{}: "{}"'.format(self.message, self.expression)
```

Reply:
Hi @bfahr - good idea; done!